serverless/cmd/client/client.go

```diff
@@ -0,0 +1,124 @@
+// Copyright 2021 Google LLC. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package main is a read-only client command for interacting with serverless logs.
```

Comment:
In main files its more standard to say the name of the tool and give example usage. See https://github.com/google/trillian-examples/blob/master/binary_transparency/firmware/cmd/flash_tool/flash_tool.go#L15 for a good example.

Reply:
done, ta